docs/pandas.rst

```diff
@@ -0,0 +1,333 @@
+.. _pandas:
+
+Pandas support
+==============
+
+It is convenient to use the `Pandas package`_ when dealing with numerical data, so Pint provides `PintArray`. A `PintArray` is a `Pandas Extension Array`_, which allows Pandas to recognise the Quantity and store it in Pandas DataFrames and Series.
+
+For this to work, we rely on `Pandas Extension Types`_ which are still experimental. As a result, we currently have to build the latest version of Pandas' master branch from source as documented in the `Pandas README`_.
+
+Basic example
+-------------
+
+This example gives you the basics, but is slightly fiddly as you are not reading from a file. A more normal use case is given in `Reading a csv`_.
+
+To use Pint with Pandas, as stated above, firstly ensure that you have the latest version of Pandas installed. Then import the relevant packages and create an instance of a Pint Quantity:
+
+.. doctest::
+
+   >>> import pandas as pd
+   >>> import numpy as np
+   >>> import pint
+   >>> from pint.pandas_interface import PintArray
+
+   >>> ureg = pint.UnitRegistry()
+   >>> Q_ = ureg.Quantity
+
+.. testsetup:: *
+
+   import pandas as pd
+   import numpy as np
+   import pint
+   from pint.pandas_interface import PintArray
+
+   ureg = pint.UnitRegistry()
+   Q_ = ureg.Quantity
+
+Next, we can create a DataFrame with PintArray's as columns
+
+.. doctest::
+
+   >>> torque = PintArray(Q_([1, 2, 2, 3], "lbf ft"))
+   >>> angular_velocity = PintArray(Q_([1000, 2000, 2000, 3000], "rpm"))
+   >>> df = pd.DataFrame({"torque": torque, "angular_velocity": angular_velocity})
+   >>> print(df)
+                       torque             angular_velocity
+   0  1 foot * force_pound  1000 revolutions_per_minute
+   1  2 foot * force_pound  2000 revolutions_per_minute
+   2  2 foot * force_pound  2000 revolutions_per_minute
+   3  3 foot * force_pound  3000 revolutions_per_minute
+
+Operations with columns are units aware so behave as we would intuitively expect
+
+.. doctest::
+
+   >>> df['power'] = df['torque'] * df['angular_velocity']
+   >>> print(df)
+                    torque             angular_velocity  \
+   0  1 foot * force_pound  1000 revolutions_per_minute
+   1  2 foot * force_pound  2000 revolutions_per_minute
+   2  2 foot * force_pound  2000 revolutions_per_minute
+   3  3 foot * force_pound  3000 revolutions_per_minute
+
+                                                 power
+   0  1000 foot * force_pound * revolutions_per_minute
+   1  4000 foot * force_pound * revolutions_per_minute
+   2  4000 foot * force_pound * revolutions_per_minute
+   3  9000 foot * force_pound * revolutions_per_minute
+
+Data accessing is a little bit awkward (fixing this would be helpful), but can be done as shown below
```

Comment:
you could register an accessor to do this 
eg

df.power.pint.to()

becomes possible

Reply:
ok sounds sensible (if @andrewgsavage can do it as this is his domain).

My comment was more directed at the fact that you have to do `df.power.values.data` to get your pint array back rather than just `df.power.values` but maybe that can't be helped

Reply:
df.power.values should be a PintArray otherwise your impl is incorrect

Reply:
ok then I guess it has to stay like this

Reply:
> df.power.pint.to()

Yea had series accessors in there, but not documented/tested. They're now in the docs and in a PR with tests.